src/components/Table/Table.stories.tsx

```diff
@@ -176,102 +229,162 @@ const Template: ComponentStory<typeof Table> = (args) => {
+        </TableHeader>
+        <TableBody>
+          {rows
+            .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
+            .map((row) => (
+              <TableRow
+                key={row.Saksnr}
+                rowData={{ Saksnr: row.Saksnr }}
+              >
+                {args.selectRows && (
+                  <TableCell radiobutton={true}>
+                    <RadioButton
+                      name={row.Saksnr}
+                      onChange={(event) => handleRadioButton(event)}
+                      value={row.Saksnr}
+                      checked={checkSelectedValue(row)}
+                      label={row.Saksnr}
+                      hideLabel={true}
+                    ></RadioButton>
+                  </TableCell>
+                )}
+                <TableCell>{row.Saksnr}</TableCell>
+                <TableCell>{row.Produkt}</TableCell>
+                <TableCell>{row.Status}</TableCell>
+                <TableCell>
+                  <img
+                    className={cn(classes['checkmark'])}
+                    src={row.BildeSrc}
+                    alt={row.BildeAlt}
+                  ></img>
                 </TableCell>
-              )}
-              <TableCell>{row.applicationNr}</TableCell>
-              <TableCell>{row.product}</TableCell>
-              <TableCell>{row.status}</TableCell>
-              <TableCell>
-                <img
-                  className={cn(classes['checkmark'])}
-                  src={row.imageSrc}
-                  alt={row.imageAlt}
-                ></img>
-              </TableCell>
-            </TableRow>
-          ))}
-      </TableBody>
-      <TableFooter>
-        <TableRow>
-          <TableCell colSpan={5}>
-            <Pagination
-              numberOfRows={rows.length}
-              rowsPerPageOptions={[5, 10, 15, 20]}
-              rowsPerPage={rowsPerPage}
-              onRowsPerPageChange={handleChangeRowsPerPage}
-              currentPage={page}
-              setCurrentPage={handleChangeInCurrentPage}
-              descriptionTexts={description}
-            />
-          </TableCell>
-        </TableRow>
-      </TableFooter>
-    </Table>
-  );
+              </TableRow>
+            ))}
+        </TableBody>
+        <TableFooter>
+          <TableRow>
+            <TableCell colSpan={5}>
+              <Pagination
+                numberOfRows={rows.length}
+                rowsPerPageOptions={[5, 10, 15, 20]}
+                rowsPerPage={rowsPerPage}
+                onRowsPerPageChange={handleChangeRowsPerPage}
+                currentPage={page}
+                setCurrentPage={handleChangeInCurrentPage}
+                descriptionTexts={description}
+                screenSize={ScreenSize.Laptop}
+              />
+            </TableCell>
+          </TableRow>
+        </TableFooter>
+      </Table>
+    );
+  };
+
+  return <>{screenSize === ScreenSize.Mobile ? isMobile() : isLaptop()}</>;
```

Comment:
Commenting here to say the same thing as in the `app-frontend-react` review. I don't particularly like for us to suggest to developers that they make a global switch/case and copy-paste all their code in order to implement a mobile/desktop view. Having to just call the component once (and have it figure out how to render the input in mobile/desktop views) looks a lot nicer than having duplicate lines of code.